common/thorhelper/thorcommon.hpp

```diff
@@ -32,6 +32,7 @@
 static unsigned const defaultDaliResultOutputMax = 2000; // MB
 static unsigned const defaultDaliResultLimit = 10; // MB
 static unsigned const defaultMaxCsvRowSize = 10; // MB
+extern jlib_decl const cycle_t defaultFlushTimerInterval;
```

Comment:
I told you the wrong qualifier.  It needs to be:

`extern THORHELPER_API const cycle_t defaultFlushTimerInterval;
`